src/main/pages/projects.page.js

```diff
@@ -0,0 +1,73 @@
+const { I, testData } = inject();
+
+const searchProjectBox = { css: '.ant-input' }
+const projecFortressName = { css: '.styled[href="/projects/fortress"]'}
+
+const lblTitleProject = { css: '.ant-typography' }
+
+const projectName = { css: '#name' }
+const status = { css: '#status' }
+const country = '#countryID'
+const accountManagers = '#accountManagers_0_employeeID'
+const deliveryManagers = '#deliveryManagers_0_employeeID'
+const projectEmail = '#projectEmail'
+const clientEmail = '#clientEmail_0'
+const projectType = '#type'
+const projectFunction = '#function'
+const developmentProjectFunction = '.ant-select-item.ant-select-item-option.ant-select-item-option-active[title="Development"]'
+const submitBtn = 'button[type="submit"]'
+
+const name = 'AUTO PROJECT ' + I.randomString(5, 'aA');
+
+class ProjectsPage {
+    then_searchProject() {
+        I.click(searchProjectBox)
+        I.fillField(searchProjectBox, 'Not exist project')
+        I.see('No Data')
+        I.click(searchProjectBox)
+        I.clearField(searchProjectBox)
+        I.fillField(searchProjectBox, 'Fortress')
+        I.see('Fortress', projecFortressName)
+    }
+    
+
+    then_clickOnCreateNewProject() {
```

Comment:
should be when_clickOnCreate... because this is the action we need to take. using then_... when verifying something @nganlk-setel

Reply:
Hi @nhuthm-setel updated